packages/cli-hydrogen/src/cli/services/deploy.ts

```diff
@@ -0,0 +1,20 @@
+import {DeployConfig} from './deploy/types.js'
+import {getDeployConfig} from './deploy/config.js'
+import {createDeploymentStep, runBuildCommandStep} from './deploy/deployer.js'
+import {output} from '@shopify/cli-kit'
+
+export async function deployToOxygen(_config: DeployConfig) {
+  const config = await getDeployConfig(_config)
+  // eslint-disable-next-line no-console
```

Comment:
`console.log`'s are discouraged and also disabling the rule. If you want to output something for the user use `output.info` or `output.debug` if you want to make it opt-in via the `--verbose` flag.